src/lib/constants/quiz.ts

```diff
@@ -0,0 +1,98 @@
+export interface QuizData {
+  questions: Question[];
+}
+
+interface Question {
+  prompt: string;
+  choices: Choice[];
+}
+
+interface Choice {
+  content: string;
+  // Where color is a valid HTML color
+  color: string;
+  // Where the choice is an index number for use in quiz.svelte
+  // ai => 0, dev => 1, design => 2, algo => 3, N/A => 4
+  match: string;
+}
+
+export const QUIZ_DATA: QuizData = {
+  questions: [
+    {
+      prompt: "What's your favorite color",
+      choices: [
+        {
+          content: 'ai',
+          color: '#21D19F',
+          match: 'aiChoice',
```

Comment:
Implement enum `TeamMatch` in `/src/lib/constants/quiz.ts`.
```suggestion
          match: TeamMatch.AI,
```

Reply:
Done